source/fundamentals/serialization/poco.txt

```diff
@@ -203,26 +250,303 @@ The following code sample maps the ``Identifier`` property to the
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 5
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(DateOnly = true)]
+       public DateTime DateOfBirth { get; set; }
+   }
+
+You can also use the ``[BsonDateTimeOptions()]`` attribute to specify the ``DateTimeKind``
+of a ``DateTime`` property. In the following code example, the ``PatientRecord`` class
+uses a ``DateTime`` for the ``AppointmentTime`` property.
+The ``[BsonDateTimeOptions(Kind = DateTimeKind.Local)]`` attribute indicates
+that the time component of the property's value is in local time. When the
+driver serializes this property, it will convert the time to UTC, the standard
+format for times stored in MongoDB.
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 8
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(DateOnly = true)]
+       public DateTime DateOfBirth { get; set; }
+
+       [BsonDateTimeOptions(Kind = DateTimeKind.Local)]
+       public DateTime AppointmentTime { get; set; }
+   }
+
+You can also specify one or both of the previous ``DateTime`` options when
+registering the class map:
+
+.. code-block:: csharp
+   :copyable: true
+
+   BsonClassMap.RegisterClassMap<House>(classMap => 
+   {
+      classMap.AutoMap();
+      classMap.MapMember(p => p.DateOfBirth).SetSerializer(new DateTimeSerializer(dateOnly: true));
+      classMap.MapMember(p => p.AppointmentTime).SetSerializer(new DateTimeSerializer(DateTimeKind.Local));
+   });
+
+.. tip:: DateTimeKind Values
+
+   The ``DateTimeKind`` enum is part of the {+framework+}. For more information on
+   its members, see the `Microsoft documentation. <https://learn.microsoft.com/en-us/dotnet/api/system.datetimekind?view=net-8.0>`__
+
+Customize Dictionary Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``DictionaryRepresentation`` enum defines the formats the {+driver-short+}
+can serialize a ``Dictionary`` to. This enum includes the following members:
+
+- **Document**: Default. The driver serializes the ``Dictionary`` to a ``BsonDocument``.
+  Each entry in the dictionary is a ``BsonElement`` with a name equal to
+  the entry's key and a value equal to the entry's value. You can
+  use this representation only when all the keys in the dictionary are strings that
+  are also valid ``BsonElement`` names.
+- **ArrayOfArrays**: The driver serializes the dictionary to a ``BsonArray``. Each
+  entry in the dictionary is a nested ``BsonArray`` that contains two elements:
+  the entry's key and the entry's value.
+- **ArrayOfDocuments**: The driver serializes the dictionary to a ``BsonArray``. Each
+  entry in the dictionary is a nested ``BsonDocument`` of the form
+  ``{ k : key, v : value }``. Because the keys and values are
+  tagged with element names, this format is much easier to query than ``ArrayOfArrays``.
+
 Example
```

Comment:
S: as part of this ticket, you could edit the example to include some of these new customizations, such as the ID generator or something!

Reply:
love it. ID generator wouldn't work here, but I can customize the default values